quodlibet/po/fr.po

```diff
@@ -455,14 +447,12 @@ msgid "Cover Grid Preferences"
 msgstr "Préférences du navigateur à panneaux"
 
 #: ../quodlibet/browsers/covergrid/prefs.py:64
-#, fuzzy
 msgid "Show album _text"
-msgstr "Montrer les _jaquettes des albums"
+msgstr "Montrer les _textes des albums"
 
 #: ../quodlibet/browsers/covergrid/prefs.py:71
-#, fuzzy
 msgid "Show \"All Albums\" Item"
-msgstr "Tous les albums"
+msgstr "Monter tous les éléments \"Tous les albums\""
```

Comment:
"Monter" -> "Montrer"

I need to have a look at this message in its context, but it does sound weird like that. The message in English does not talk about multiple items, so I would go with "Montrer l'élément \"Tous les albums\"", but…